lib/api.js

```diff
@@ -5,6 +5,7 @@
 
 "use strict";
 
+require("v8-compile-cache"); // to use V8's code cache to speed up instantiation time
```

Comment:
I'm worried in this because this can have side effects for applications that use ESLint Node.js API. I think that we should not force application authors using cache.